www/components/sidebar.tsx

```diff
@@ -94,17 +96,32 @@ export default function Sidebar({
     const conversation = await api?.new();
     postHog?.capture('user_created_conversation');
     setConversationId(conversation?.conversationId);
-    mutateConversations([conversation, ...conversations]);
+    mutateConversations([conversation!, ...conversations]);
   }
 
+  const fetchUser = async () => {
+    const {
+      data: { user },
+    } = await supabase.auth.getUser();
+    return user;
+  };
+
+  const {
+    data: user,
+    error: userError,
+    isLoading: isUserLoading,
+  } = useSWR('user', fetchUser);
+
   return (
     <div
-      className={`fixed lg:static z-20 inset-0 flex-none h-full w-full lg:absolute lg:h-auto lg:overflow-visible lg:pt-0 lg:w-60 xl:w-72 lg:block lg:shadow-lg border-r border-gray-300 dark:border-gray-700 ${isSidebarOpen ? '' : 'hidden'
-        }`}
+      className={`fixed z-20 inset-0 flex-none h-full w-full lg:absolute lg:h-auto lg:overflow-visible lg:pt-0 lg:w-60 xl:w-72 lg:block lg:shadow-lg border-r border-gray-300 dark:border-gray-700 ${
+        isSidebarOpen ? '' : 'hidden'
+      }`}
     >
       <div
-        className={`h-full scrollbar-trigger overflow-hidden bg-white dark:bg-gray-950 dark:text-white sm:w-3/5 w-4/5 lg:w-full flex flex-col ${isSidebarOpen ? 'fixed lg:static' : 'sticky'
-          } top-0 left-0`}
+        className={`h-full scrollbar-trigger overflow-hidden bg-white dark:bg-gray-950 dark:text-white sm:w-3/5 w-4/5 lg:w-full flex flex-col ${
```

Comment:
<img width="894" alt="image" src="https://github.com/user-attachments/assets/e908251f-f457-4e66-9732-31e299043210">

Text in sidebar is white in dark mode, makes it hard to read, should stay black the entire time